alpine-rails-nginx/Dockerfile

```diff
@@ -15,44 +13,28 @@ RUN echo '@testing http://dl-4.alpinelinux.org/alpine/edge/testing' >> /etc/apk/
   | xargs apk add --update-cache \
   && rm -- /tmp/apk-packages
 
-# Install nginx
-COPY install-nginx.sh /tmp/
-COPY nginx-remove-server-name.patch /tmp/
-RUN /tmp/install-nginx.sh \
-  && rm -rf -- /usr/local/src /tmp/*
-
-# Upgrade bundler
-RUN echo 'gem: --no-document' > ~/.gemrc \
-  && gem install bundler
-
-# Add utility scripts
-COPY setup-directories.sh /root/
-
-# Setup system configuration
-COPY etc /etc/
+# Create system users
+RUN adduser -DSH -G nogroup nginx \
+  && adduser -DSH -G nogroup rails
```

Comment:
Its IMO the other way round. Via sharing the primary group, we risk to miss a group permission setting. Via not sharing the group we have another layer of defense.

Also we can never know which 3rd party library fails to kill group permissions when writing stuff to `/tmp` or other shared locations.